src/install.rs

```diff
@@ -185,6 +185,7 @@ fn write_disk(
             "boot",
             false,
             mount::MsFlags::empty(),
+            Some("/boot"),
```

Comment:
We don't own `/boot` and can't mount an arbitrary filesystem on it.  For example, coreos-installer might be running on an arbitrary Linux distro on bare metal, to set up the virtual disk for a FCOS VM.  If this code raced with the installation of OS updates on the host, both the host and guest `/boot` filesystems could be corrupted.  That's why we mount `/boot` onto a temporary mountpoint.

Aside from simplifying zipl invocation, how is this change needed for #533?

Reply:
running `zipl` without any options requires `/boot` , otherwise we'd have to specify kernel, initrd,... . 
what if we make mounting of `/boot`  s390x-arch dependent?